tests/lib/rules/space-infix-ops.js

```diff
@@ -32,15 +32,26 @@ ruleTester.run("space-infix-ops", rule, {
         { code: "function foo(a = 0) { }", parserOptions: { ecmaVersion: 6 } },
         { code: "function foo(a: number = 0) { }", parser: path.resolve(__dirname, "../../fixtures/parsers/flow-stub-parser.js"), parserOptions: { ecmaVersion: 6 } },
         { code: "a ** b", parserOptions: { ecmaVersion: 7 } },
+        { code: "a ** b ** c", parserOptions: { ecmaVersion: 7 } },
+        { code: "(-a) ** b ** c", parserOptions: { ecmaVersion: 7 } },
+        { code: "-(a ** b ** c)", parserOptions: { ecmaVersion: 7 } },
+        { code: "a**b", parserOptions: { ecmaVersion: 7 }, options: [{ "**": "never" }] },
+        { code: "a**b**c", parserOptions: { ecmaVersion: 7 }, options: [{ "**": "never" }] },
+        { code: "(-a)**b**c", parserOptions: { ecmaVersion: 7 }, options: [{ "**": "never" }] },
+        { code: "-(a**b**c)", parserOptions: { ecmaVersion: 7 }, options: [{ "**": "never" }] },
         { code: "a|0", options: [{ int32Hint: true }] },
-        { code: "a |0", options: [{ int32Hint: true }] }
+        { code: "a |0", options: [{ int32Hint: true }] },
+        { code: "a + b-c * d*e", options: [{ "+": "always", "-": "never", "*": "ignore" }] },
+        { code: "a + b-c * d*e", options: [{ all: "always", "+": "always", "-": "never", "*": "ignore" }] },
+        { code: "a + b-c * d*e", options: [{ all: "never", "+": "always", "-": "never", "*": "ignore" }] },
+        { code: "a + b-c * d*e", options: [{ all: "ignore", "+": "always", "-": "never", "*": "ignore" }] }
     ],
     invalid: [
         {
             code: "a+b",
             output: "a + b",
             errors: [{
-                message: "Infix operators must be spaced.",
+                message: "+ operator must be surrounded by spaces.",
```

Comment:
I think we should surround the operator in question with a delimiter of some sort. My preference would be backtick, but I wouldn't mind single or double quotes either.

I also think `"Operator \`+\` must be surrounded by spaces."` would be slightly more readable, just to avoid the message starting with an operator or quotes. But I'll defer to consensus on this point.

Obviously, any changes made would apply to all invalid tests.